lib/mobility-core/src/Kernel/Types/Cac.hs

```diff
@@ -31,3 +45,31 @@ data CACData = CACData
     variantIds :: Text
   }
   deriving (Show, Eq, Generic, ToJSON, FromJSON)
+
+getToss :: (CacheFlow m r) => Maybe Text -> m Int
+getToss srId = do
+  gen <- newStdGen
+  expTime <- fromIntegral <$> asks (.cacheConfig.configsExpTime)
```

Comment:
Lets check if this expiry is proper. And in case of Rentals or intercity, ride could go for 7 days too.